shared/src/libs/enums/upload-file-content-type-to-extensions.ts

```diff
@@ -0,0 +1,9 @@
+import { ContentType } from './content-type.enum.js';
```

Comment:
let's remove this enum

Reply:
@Liza-Veis Should I show the user content type like `image/...`? I thought a have to use extensions only

Reply:
we can use extensions for both validation messages

Reply:
@Liza-Veis I know, but in the backend file upload plugin I have to use content types to validate file type, so I need both extensions and content types

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/8ed3b242-38c5-4140-9463-f20e4d79dbc6)

Reply:
something like this would be okay `extensions: Object.values(UploadFileExtension)`

Reply:
@Liza-Veis It won't work, because we're comparing it with a content type

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/1625da05-0e10-47fe-b00b-3af72b278a8a)

Reply:
ok, let's leave it